script.module.simpleeval/addon.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.simpleeval" name="Simple Eval" version="0.0.1" provider-name="SiLVO">
```

Comment:
Versioning is wrong. Which version is this? Please use upstream versions: https://pypi.org/project/simpleeval/#history

Reply:
I have no idea which version this is. How do I find that out?

Reply:
@enen92 I did not create this script, I am just maintaining it. I do not know which version of simpleeval is being used ATM.

Reply:
Hmmm... I think I am just gonna update the code and use the latest version

Reply:
yeah that's the way to go. Update to latest upstream version and check if your script needs any adaption :)

Reply:
Everything looks good after testing so I think I'm good 🤞